packages/q/qt5base/xmake.lua

```diff
@@ -206,13 +206,12 @@ package("qt5base")
         os.rmdir(path.join(installdir, version))
 
         package:data_set("qt", qt_table(installdir, version:shortstr()))
-        print("xxxxxxxxxxx")
-        print(os.files(path.join(package:installdir(), "**", "moc")))
-        print(os.files(path.join(package:installdir(), "**", "QCoreApplication")))
     end)
 
     on_test(function (package)
         local qt = assert(package:data("qt"))
-        os.vrun(path.join(qt.bindir, "moc") .. " -v")
-        os.vrun(path.join(qt.bindir, "rcc") .. " -v")
+        if not package:is_plat("mingw") then
```

Comment:
This is actually fine on mingw@msys and mingw@windows

But I'm wondering if a qt package makes sense for cross-compilation, since it means Qt tools can't be called

Reply:
I remember qt's official mingw sdk on macos, moc can be called.